### ISSUE TITLE
/views/instances/{key} page is not accessible for screen reader users(blind and visually impaired)

### ISSUE THREAD

**Issue by hamidrezaabroshan** [camunda-community-hub/zeebe-simple-monitor#334]
I'm a blind java developer and I wanted to see which work flow is executed and in which flow has errors.
My colleagues can see these details from this page but because this page used a diagram that made by canvas, I can not use this.
I looked through the code and this view returns this html file.
instance-detail-view.html and I think this html uses this component: bpmn-diagram.html.
To make it accessible, search for making canvas accessible for screen readers and see  <a href="https://stevefaulkner.github.io/Articles/Notes%20on%20accessibility%20of%20text%20replacement%20using%20HTML5%20canvas.html"> this link also</a>
Note: I'm using windows 10 and free and open source Nvda screen reader.
It is portable and after running and accepting agreement, you can install or just press continue running and test the behaviour of Simple monitor app.
Thanks.

**Comment by oleschoenburg** [comment]
Hey @hamidrezaabroshan, thanks for taking the time to report this issue. We delegate the rendering to bpmn.io so I think it might be best to fix this upstream, here: https://github.com/bpmn-io/bpmn-js. That way all other applications using bpmn.io could benefit as well. 

If I understand correctly, one simple fix could be to add textual information inside the `<canvas>` tag. Could you create a new issue [here](https://github.com/bpmn-io/bpmn-js/issues) and document what information you would like to have available? Thanks in advance :slightly_smiling_face:

**Comment by hamidrezaabroshan** [comment]
Hi @oleschoenburg,
I filed this issue <a href="https://github.com/bpmn-io/bpmn-js/issues/1548"> here</a>.
When they get  an appropriate answer,
I'll close this issue here.
Thanks.

**Comment by saig0** [comment]
I'm closing this issue because of inactivity. Feel free to reopen if the issue is still relevant.
